src/routes/safe/components/Apps/components/AddAppForm/FormButtons.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button, Divider } from '@gnosis.pm/safe-react-components'
+import React, { ReactElement, useMemo } from 'react'
+import { useFormState } from 'react-final-form'
+import styled from 'styled-components'
+
+import GnoButton from 'src/components/layout/Button'
+import { SafeApp } from 'src/routes/safe/components/Apps/types.d'
+import { isAppManifestValid } from 'src/routes/safe/components/Apps/utils'
+
+const StyledDivider = styled(Divider)`
+  margin: 16px -24px;
+`
+
+const ButtonsContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+
+interface Props {
+  appInfo: SafeApp
+  onCancel: () => void
+}
+
+const FormButtons = ({ appInfo, onCancel }: Props): ReactElement => {
```

Comment:
So from now on are we going to deconstruct ReactElement from React?

Just asking because usually we are using it as React.ReacElement.

Reply:
It was a suggestion from @fernandomg because the new React version will no allow to access it from `React.`

Reply:
:grin: